resources/LinuxBuild.cmake

```diff
@@ -12,21 +12,20 @@ set (DAKOTA_PYTHON ON CACHE BOOL "python interface on" FORCE)
 # DO NOT ENABLE!!! set (MPI_DEBUG OFF CACHE BOOL "mpi debug off" FORCE)
 
 ### no mpi
+set( DAKOTA_HAVE_MPI FALSE
```

Comment:
Maybe we should add a comment here that this file should be shared between the building of `dakota` itself and Python wrappers in Carolina.